recipes/pangeo-forge-recipes-feedstock/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pangeo-forge-recipes" %}
+{% set version = "0.8.2" %}
+{% set hash_val = "63512b9fe2898332a6293e60bb36e7c3e5e2643cc52767e3622a051a738c154f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_val }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.8
+    - dask
+    - distributed
+    - fsspec[http] >=2021.6.0
```

Comment:
Sqaure brackets do something else in conda than pip. They probably have no effect here.